UI/static/css/landingpage.css

```diff
@@ -0,0 +1,77 @@
+body{
+    background-image: url(../images/road.jpg);
+    background-size: cover;
+}
+
+.slogan{
+    clear: both;
+    text-align: center;
+}
+
+.intro{
+    position: relative;
+    margin: auto;
+    margin-left: 5%;
+    height: auto;
+    width: 35%;
+    display: inline-block;
+    vertical-align: top;
+}
+
+.intro>h1{
+    font-size: 40px;
+    font-weight: 900;
+    align-content: center;
+}
+
+.intro>p{
+    text-align: center;
+    font-size: 24px;
+}
+
+.signupform{
+    height: 60%;
+    width: 35%;
+    background-color: cyan;
+    display: inline-block;
+    margin-left: 20%;
+    vertical-align: middle;
+    align-content: center;
+}
+
+.call-to-action{
```

Comment:
Great job using classes to style elements. Makes it easy to know the group of elements that belong together